app/facades/elections_facade.rb

```diff
@@ -0,0 +1,34 @@
+#General elections and referendums are considered to be the same in civic service
+#each array has nil values for the election that dosen't match
+# Serializer can filter out nil values
+class ElectionsFacade
+
+  def self.elections(location)
+    service = ElectionService.get_elections(location)[:contests]
+
+    all_elections = service.map do |data|
+        if data[:type] != 'Referendum'
+            Election.new(data)
+        end
+    end
+    elections = all_elections.reject {|elec| elec == nil}
+    return elections
```

Comment:
Returns are implicit - Don't need to specifically say 'return elections', can just say 'elections' instead

Reply:
Nice, good call.